src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -83,7 +99,7 @@ def get(self, request: Request, project) -> Response:
                 scope.set_tag("integration_provider", provider)
                 scope.set_tag("stacktrace_link.platform", platform)
 
-                if not filepath.startswith(config.stack_root):
+                if not filepath.startswith(config.stack_root) and not frame:
```

Comment:
Just linking to the discussion we had on the other pr here for @getsentry/ecosystem's advice https://github.com/getsentry/sentry/pull/35349#discussion_r890565081